src/ethereum/EnigmaContractReaderAPI.js

```diff
@@ -333,21 +334,21 @@ class EnigmaContractReaderAPI {
       });
     });
   }
+
   /**
    * * Get Ethereum block number
    * @return {Promise} returning {Integer} : blockNumber
    * */
-  getEthereumBlockNumber() {
-    return new Promise((resolve, reject) => {
-      this._web3.eth.getBlockNumber((error, data) => error ? reject(error) : resolve(data));
-    });
+  async getEthereumBlockNumber() {
```

Comment:
Because other parts of the code uses it from here. Those other parts doesn't have access to a `web3` instance which is needed in order to call the `util` function.  
I didn't want to try to get a `web3` instance there, but as I'm writing it I've remembered I can just do `api.w3()` :man_facepalming: :man_facepalming: 

![Screenshot from 2019-11-14 10-25-04](https://user-images.githubusercontent.com/22766533/68839466-6faca500-06c9-11ea-8d34-cfa1cf0b4e64.png)

Will fix.